src/memgraph.cpp

```diff
@@ -754,59 +756,87 @@ class AuthQueryHandler final : public memgraph::query::AuthQueryHandler {
     }
   }
 
-  void GrantPrivilege(const std::string &user_or_role,
-                      const std::vector<memgraph::query::AuthQuery::Privilege> &privileges,
-                      const std::vector<std::string> &labels, const std::vector<std::string> &edge_types) override {
+  void GrantPrivilege(
+      const std::string &user_or_role, const std::vector<memgraph::query::AuthQuery::Privilege> &privileges,
+      const std::vector<std::unordered_map<memgraph::query::AuthQuery::LabelPrivilege, std::vector<std::string>>>
+          &label_privileges,
+      const std::vector<std::unordered_map<memgraph::query::AuthQuery::LabelPrivilege, std::vector<std::string>>>
+          &edge_type_privileges) override {
```

Comment:
If the same type is used / one type holds both things (conceptually), maybe a better name would be `memgraph::query::AuthQuery::LabelEdgeTypePrivilege`?